httpstan/stan_services.cpp

```diff
@@ -233,6 +233,34 @@ std::vector<double> write_array(py::dict data,
   return params_r_constrained;
 }
 
+// See exported docstring
+std::vector<double> transform_inits(py::dict data,
+                                    py::dict constrained_parameters) {
+  std::vector<double> params_r_unconstrained;
+  stan::io::array_var_context &var_context = new_array_var_context(data);
+  // random_seed, the second argument, is unused but the function requires it.
+  stan::model::model_base &model = new_model(var_context, (unsigned int)1, &std::cout);
+  stan::io::array_var_context &param_var_context = new_array_var_context(constrained_parameters);
+  // unconstrain parameters from their defined support
+  std::exception_ptr p;
+  std::vector<int> params_i(model.num_params_i(), 0);
+  try {
+    // params_i, the third argument, is unused but the function requires it (see model_base.hpp).
+    model.transform_inits(param_var_context, params_r_unconstrained, params_i, &std::cout);
```

Comment:
If transform_inits can throw an exception, a comment reminding us why might be helpful here. (I find myself wondering the same thing for write_array and log_prob (with unconstrained params), actually.)

Reply:
Hmm.. `log_prob` throws an exception if the evaluation fails (you can see this behaviour in Stan's initialisation method). I'm not 100% sure why write_array/transform_inits would fail, but have seen this mentioned on the discourse/main Stan repo which is why I added the try just to be safe.